src/Controller/CRUDController.php

```diff
@@ -656,6 +692,23 @@ public function showAction($id = null)
             throw $this->createNotFoundException(sprintf('unable to find the object with id: %s', $id));
         }
 
+        if ($parentAdmin = $this->admin->getParent()) {
```

Comment:
Having this code duplicated 3 times doesnt sound great, maybe we can extract a method here?

Reply:
Message is different, but I guess I can pass that as an argument

Reply:
Or use a single copy, but I cant give you a good example right now, will think about it.

Reply:
better?

Reply:
I guess this is it.